building/build-debs/common/package-build-helpers.sh

```diff
@@ -7,9 +7,23 @@ then
 	exit 1
 fi
 
+if [ -z "${APT_REPOSITORY:-}" ]
+then
+	echo "Need to specify apt repository:"
+	echo "\$ export APT_REPOSITORY=<username>/<branch>"
```

Comment:
You can use single quotes for this, instead of escaping.

Reply:
Thanks! Forgot you could do that.